src/test/java/seedu/address/storage/JsonSerializableAddressBookTest.java

```diff
@@ -23,8 +23,8 @@ public void toModelType_typicalEateriesFile_success() throws Exception {
         JsonSerializableAddressBook dataFromFile = JsonUtil.readJsonFile(TYPICAL_EATERIES_FILE,
                 JsonSerializableAddressBook.class).get();
         AddressBook addressBookFromFile = dataFromFile.toModelType();
-        AddressBook typicalEaterysAddressBook = TypicalEateries.getTypicalAddressBook();
-        assertEquals(addressBookFromFile, typicalEaterysAddressBook);
+        AddressBook typicalEateriesAddressBook = TypicalEateries.getTypicalAddressBook();
```

Comment:
why u do this what did u even change HAHAHA

Reply:
typicalEaterysAddressBook to typicalEateriesAddressBook

Reply:
oh lolsies